### ISSUE TITLE
[Web/Webgpu]  WebGPU None-zero status code returned while running Softmax node.

### ISSUE THREAD

**Issue by EvoDmiK** [microsoft/onnxruntime#18314]
### Describe the issue

Hello, I try to convert pytorch model to onnx model.
by the way, error occurred when neural network archtecture includes softmax layer.
like below log lines.

[[repository for my pytorch model]](https://github.com/swz30/MIRNetv2)

```js
wasm-utils.ts:60 Uncaught (in promise) Error: failed to call OrtRun(). ERROR_CODE: 1, ERROR_MESSAGE: Non-zero status code returned while running Softmax node. 
Name:'/body/body.0/body/body.0/skff_mid/softmax/Softmax' Status Message: Failed to run JSEP kernel
```

below image is entire log lines
![log_lines](https://github.com/microsoft/onnxruntime/assets/93193661/c0db110a-ebc3-4a20-8648-c8c73a7b25fc)

### To reproduce

run following code in a browser that supports WebGPU
- import necessary modules
```js
        <script async src="https://docs.opencv.org/4.x/opencv.js"></script>
	<script src="https://code.jquery.com/jquery-1.11.3.min.js"></script>
	<script src="https://cdn.jsdelivr.net/npm/mathjs@11.8.0/lib/browser/math.min.js"></script>
        <script src="https://{my private ip}/templates/static/dist/ort.webgpu.min.js"></script>
```
- create inference session
```js
    const model_path = "https://{my private ip}/templates/static/models/lle_model.onnx";
    const options = {
					    executionProviders  : ["webgpu"],
					    graphOptimizationLevel : "extended",
					    executionMode          : "parallel"
				    };
    
    const session    = ort.InferenceSession.create(model_path, options);
    console.log("model load success.");
```

- and inference function for my session
```js
        session.then(
            sess => {

                console.log("Inference started.")
                const feeds = {image : rTensor};
                console.time("inference");

                sess.run(feeds).then( pred => {
                    to_html(pred.enhanced_image);
                });

                console.timeEnd("inference");
                console.log("Inference complete.")
            }
        );
```

### ONNX Runtime Installation

Built from Source

### ONNX Runtime Version or Commit ID

onnxruntime-webgpu 1.16.0

### Execution Provider

'webgpu' (WebGPU)
```[tasklist]
### Tasks
```

**Comment by guschmue** [comment]
looking

**Comment by guschmue** [comment]
kernel "[Softmax] /body/body.0/body/body.0/skff_mid/softmax/Softmax" is called with axis=1, shape=[1,2,120,1,1]
 and the kernel only implements support for the last axis.

**Comment by EvoDmiK** [comment]
thanks for your reply!

**Comment by EvoDmiK** [comment]
thanks for your reply!

**Comment by EvoDmiK** [comment]
thanks for your reply!

**Comment by guschmue** [comment]
we can see if we can ass support for axis=1

**Comment by EvoDmiK** [comment]
> we can see if we can ass support for axis=1

thank you for your reply. and I solved this problem using onnx quantization.

**Comment by chopin1998** [comment]
> > we can see if we can ass support for axis=1
> 
> thank you for your reply. and I solved this problem using onnx quantization.

what's your mean? could you please show some code?
